src/Compilers/CSharp/Portable/PublicAPI.Shipped.txt

```diff
@@ -301,11 +301,8 @@ Microsoft.CodeAnalysis.CSharp.Syntax.ArgumentListSyntax.WithOpenParenToken(Micro
 Microsoft.CodeAnalysis.CSharp.Syntax.ArgumentSyntax
 Microsoft.CodeAnalysis.CSharp.Syntax.ArgumentSyntax.Expression.get -> Microsoft.CodeAnalysis.CSharp.Syntax.ExpressionSyntax
 Microsoft.CodeAnalysis.CSharp.Syntax.ArgumentSyntax.NameColon.get -> Microsoft.CodeAnalysis.CSharp.Syntax.NameColonSyntax
-Microsoft.CodeAnalysis.CSharp.Syntax.ArgumentSyntax.RefOrOutKeyword.get -> Microsoft.CodeAnalysis.SyntaxToken
```

Comment:
This is a breaking change to remove this from shipped. The PublicAPI analyzer has a **REMOVED** syntax you can use in PublicAPI.Unshipped.txt to express this.